gulpfile.js

```diff
@@ -0,0 +1,32 @@
+var gulp = require('gulp');
+var zip = require('gulp-zip');
+var del = require('del');
+var runSequence = require('run-sequence');
+var path = require('path');
+var buildDest = path.resolve("./build");
+
+var srcFiles = path.resolve('./src/**/*.*');
+var name = require('./package.json').name;
+
+gulp.task('remove-build-zip', function(){
+  del.sync([buildDest + "/" + name + '.zip']);
+});
+
+gulp.task('zip-build', function(){
+  return gulp.src(buildDest + '/**/*')
+    .pipe(zip(name + '.zip'))
+    .pipe(gulp.dest(buildDest));
+});
+
+gulp.task('add-src', function(){
```

Comment:
build?

Reply:
to keep extensions similar it should probably get a similar webpack setup, would be nice to polish and update it a bit for both of them though, maybe do an updated and polished version on this one and then apply that to the previous one aswell.

Reply:
Sounds like a good idea. Let's keep this script as work in progress until next week